ui/src/app/components/add-new-column/add-new-column.component.html

```diff
@@ -21,6 +21,28 @@ <h2>Column Details</h2>
             </mat-form-field>
             <br>
         </div>
+        <div *ngIf="selectedDatatype != '' && autoGenMap[selectedDatatype].length > 1 && srcDbName == 'MySQL'">
```

Comment:
`MySQL` should not be hardcoded in the HTML.

Get this from a variable in the component implementation. It should also be list `supportedDbs` or some such with only one DB right now - MySQL